custom/icds_reports/reports/awc_reports.py

```diff
@@ -1086,11 +1092,23 @@ def get_status(value):
             fully_immunized_date='Yes' if row_data.fully_immunized else 'No',
             mother_name=row_data.mother_name,
             age_in_months=row_data.age_in_months,
-            nutrition_status=get_status(row_data.current_month_nutrition_status),
+            nutrition_status=get_status(
+                row_data.current_month_nutrition_status,
+                'underweight',
+                'Normal weight for age'
+            ),
             recorded_weight=row_data.recorded_weight or 0,
             recorded_height=row_data.recorded_height or 0,
-            stunning=get_status(row_data.current_month_stunting),
-            wasting=get_status(row_data.current_month_wasting),
+            stunning=get_status(
+                row_data.current_month_stunting,
+                'stunned',
```

Comment:
Should this be stunned or stunted?

Reply:
@lwyszomi

Reply:
@snopoke @sheelio Yes, should be stunted, sorry for the typo.

Reply:
I sent new commit